multimodal/setup.py

```diff
@@ -20,43 +20,41 @@
 submodule = "multimodal"
 install_requires = [
     # version ranges added in ag.get_dependency_version_ranges()
-    "numpy",
-    "scipy",
-    "pandas",
-    "scikit-learn",
-    "Pillow",
-    "tqdm",
+    "numpy",  # version range defined in `core/_setup_utils.py`
+    "scipy",  # version range defined in `core/_setup_utils.py`
+    "pandas",  # version range defined in `core/_setup_utils.py`
+    "scikit-learn",  # version range defined in `core/_setup_utils.py`
+    "Pillow",  # version range defined in `core/_setup_utils.py`
+    "tqdm",  # version range defined in `core/_setup_utils.py`
     "boto3",
     "requests",
-    "jsonschema<=4.8.0",
-    "seqeval<=1.2.2",
-    "evaluate<=0.3.0",
-    "accelerate>=0.9,<0.16",
+    "jsonschema<4.18",
+    "seqeval<1.2.3",
+    "evaluate<0.4.0",
+    "accelerate>=0.9,<0.17",
     "timm<0.7.0",
     "torch>=1.9,<1.14",
     "torchvision<0.15.0",
```

Comment:
Side note: what happened to the torchtext dependency? 🤔

Reply:
We managed to remove it by upgrading lightning... See https://github.com/autogluon/autogluon/pull/2739